### ISSUE TITLE
Magento 2.3.5 Export not working

### ISSUE THREAD

**Issue by ss-sabareesh** [magento/magento2#31027]
### Preconditions (*)

1. Magento version 2.3.5
2. More than 2 Lakh Products

### Steps to reproduce (*)

1. After clicking the continue button **"Message is added to the queue, wait to get your file soon"** but the csv is not displaying in the table also checked after an hour.
2. Also ran the CLI **bin/magento cron:run**
3. If I ran the CLI **bin/magento queue:consumers:start exportProcessor** I am getting the as **KILLED**

### Expected result (*)

1. Inside the var/export **catalog_product.csv** file should be there.
2. Need to show the product export file inside the table on the export screen

### Actual result (*)

1. Nothing is the happening table I showing blank only
2. No file present in var/export folder

**Comment by sabareesh-warlock** [comment]
give me 2.3.5-develop instance to replicate the issue

**Comment by engcom-Alfa** [comment]
Hi @sabareesh-warlock . 
Unfortunately, we are not able to reproduce this issue on fresh 2.4-develop. Everything works as expected.

**Manual testing scenario:**

**_Note:_** Tested with 200000 products

1. Run the CLI **bin/magento queue:consumers:start exportProcessor**
2. Go to Admin -> System -> Export

-  Entyty Type = Products

3. Click on the **Continue** button

"**Message is added to the queue, wait to get your file soon**"  is present

4. run in console the CLI `bin/magento cron:run`

**Actual Result:** :heavy_check_mark: The export file was successfully created

![Screenshot from 2020-11-25 11-46-06](https://user-images.githubusercontent.com/51679138/100210725-fb2f5780-2f13-11eb-9f3d-7ca66b759845.png)

:heavy_check_mark: The file present in var/export folder

![Screenshot from 2020-11-25 11-45-55](https://user-images.githubusercontent.com/51679138/100210772-084c4680-2f14-11eb-8a3f-f03efc074553.png)

 :yarn: The export file appeared only after an hour and a half as its size is more than 1 Gb ( 200000 products).

So, we have to close it.
Please feel free to comment, reopen or create new ticket according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines)
if you are still facing this issue on the latest `2.4-develop` branch. Thank you for collaboration.

**Comment by sabareesh-warlock** [comment]
@engcom-Alfa **bin/magento queue:consumers:start exportProcessor** when run this command i got the out put as killed can you suggest any reason .

![image](https://user-images.githubusercontent.com/68388960/100219319-d5b64400-2f3b-11eb-931f-80a56039cd37.png)
